### PR TITLE
Support new glyph URL syntax

### PR DESCRIPTION
**Pull request by lucaswoj**
fixes https://github.com/mapbox/mapbox-gl-style-spec/issues/309

see https://github.com/mapbox/api-assets/blob/master/lib/server.js#L59

:eyes: @jfirebaugh

### CODE REVIEW

**Comment by jfirebaugh** [comment]
This is going to conflict with d57d1ae13fb379aaa8ec89f195defd09cbbecf59 from master -- this change should win.

**Comment by jakepruitt** [comment]
Also, it should not be `/fonts/v4`, it should be `/fonts/v1`

File: js/util/mapbox.js [hunk 1]
```diff
@@ -55,7 +55,9 @@ module.exports.normalizeGlyphsURL = function(url, accessToken) {
     if (!url.match(/^mapbox:\/\//))
         return url;
 
-    return normalizeURL(url, '/v4/', accessToken);
+    var user = url.split('/')[3];
+
+    return normalizeURL('mapbox://' + user + '/{fontstack}/{range}.pbf', '/fonts/v4/', accessToken);
```
Comment (jfirebaugh) [inline js/util/mapbox.js]: Should be v1.

**Comment by lucaswoj** [comment]
Fixed `v4` --> `v1`

I guess we're going to have to deal with the merge conflict when it happens...

**Pull request closed without merging**